roles/update/tasks/main.yml

```diff
@@ -44,6 +44,51 @@
     - cifmw_ci_gen_kustomize_values_installplan_approval is defined
     - cifmw_ci_gen_kustomize_values_installplan_approval | lower == 'manual'
 
+- name: Initialize the openstack operator if needed
+  kubernetes.core.k8s:
+    kubeconfig: "{{ cifmw_openshift_kubeconfig }}"
+    api_key: "{{ cifmw_openshift_token | default(omit) }}"
+    context: "{{ cifmw_openshift_context | default(omit) }}"
+    definition: "{{ _openstack_init_resource }}"
+    state: present
+  vars:
+    _openstack_init_resource:
+      apiVersion: operator.openstack.org/v1beta1
+      kind: OpenStack
+      metadata:
+        name: openstack
+        namespace: openstack-operators
+  when:
+    - cifmw_ci_gen_kustomize_values_deployment_version is defined
+    - cifmw_ci_gen_kustomize_values_deployment_version is in ['v1.0.3', 'v1.0.6']
+
+- name: Ensure OpenStack deployment is successful and block until it is
```

Comment:
Could you update phrase to "Ensure OpenStack deployment is successful and block until it is >done<"? It looks like the sentence is not finished (at least I expect something after it is)